cirq-core/cirq/sim/state_vector_simulator.py

```diff
@@ -124,7 +124,8 @@ def __init__(
 
     @cached_property
     def final_state_vector(self) -> np.ndarray:
-        return self._get_merged_sim_state().target_tensor.reshape(-1)
+        ret = self._get_merged_sim_state().target_tensor.reshape(-1)
+        return ret / np.linalg.norm(ret)
```

Comment:
Can we raise UserWarning if `abs(norm(state_vector) - 1) > np.sqrt(np.finfo(state_vector.dtype).eps)`?

That should be sufficient for an accumulated round off from ~1/eps operations, but give some warning in case the state vector would be widely off.